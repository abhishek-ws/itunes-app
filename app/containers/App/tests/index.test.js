import React from 'react';
import { renderWithIntl } from '@utils/testUtils';
import App from '../index';
import { BrowserRouter } from 'react-router-dom';

describe('<App /> container tests', () => {
  it('should render and match the snapshot', () => {
    const { container } = renderWithIntl(
      <BrowserRouter>
        <App />
      </BrowserRouter>
    );
    expect(container).toMatchSnapshot();
  });
});
