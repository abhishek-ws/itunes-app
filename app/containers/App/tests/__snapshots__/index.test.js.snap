// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> container tests should render and match the snapshot 1`] = `
<div>
  <header
    class="ant-layout-header Header__StyledHeader-wp2jxc-0 jpfWFm"
    data-testid="header"
    intl="[object Object]"
  >
    <img
      alt="logo"
      class="Header__Logo-wp2jxc-1 bDHyGP"
      src="IMAGE_MOCK"
    />
    <p
      class="T__StyledText-gjlic1-0 kYTaGb Header__Title-wp2jxc-2 dTrOKz"
      data-testid="t"
    >
      Wednesday Solutions
    </p>
  </header>
  <main
    class="ant-layout-content"
  />
</div>
`;
